src/tests/functional/plugin/shared/src/low_precision_transformations/concat_with_intermediate_transformation.cpp

```diff
@@ -37,17 +39,25 @@ std::string ConcatWithIntermediateTransformation::getTestCaseName(const testing:
     return result.str();
 }
 
-InferenceEngine::Blob::Ptr ConcatWithIntermediateTransformation::GenerateInput(const InferenceEngine::InputInfo &info) const {
-    ngraph::element::Type netPrecision;
-    ngraph::PartialShape inputShape;
-    std::string targetDevice;
-    ngraph::pass::low_precision::LayerTransformation::Params trasformationParams;
-    bool transparentIntermediate;
-    bool multichannel;
-    std::tie(netPrecision, inputShape, targetDevice, trasformationParams, transparentIntermediate, multichannel) = this->GetParam();
+ov::test::utils::InputsMap ConcatWithIntermediateTransformation::get_input_map() {
+    auto generate_default = [](const std::shared_ptr<ngraph::Node>& node,
```

Comment:
I saw the code duplications. Also I have left one comment connected to this method... Let's discuss offline to get more info and choose the best techical solution